interface/src/app/mlmodel/resource.js

```diff
@@ -0,0 +1,9 @@
+angular.module('adage.mlmodel.resource', ['ngResource'])
+
+.factory('MLModel', ['$resource', function($resource) {
+  return $resource(
+    '/api/v0/mlmodel/'
+  );
+}])
```

Comment:
Well, it would be loosely coupled if it just lived in the other file, but @dongbohu mentioned not creating the service at all, just calling the resource inside the component code, which is why I mentioned that. Also, I think it doesn't add too much overhead inconvenience to have it in this file, and having the other file just be for angular ``components`` in the same way you might have a file for ``directives``.